lib/react.dart

```diff
@@ -590,6 +591,14 @@ abstract class Component2 implements Component {
     adapter.setStateWithUpdater(updater, callback);
   }
 
+  /// Method used to initialize the [state] of a component.
```

Comment:
As per the Dart style guide,
- https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs
- https://dart.dev/guides/language/effective-dart/documentation#prefer-using-this-instead-of-the-to-refer-to-a-members-instance

This should be something more like
```dart
/// Initializes this component's [state] to [value].
```

Reply:
Thank you for the links!! It had been a while since I had referred to those and clearly I need a refresher!